config.js

```diff
@@ -375,7 +366,7 @@ if (argv.botid || argv.bot || argv.id || argv.minrankedhandicap || argv.maxranke
         return false;
     }
 
-    if (argv.ban) {
+    if (argv.ban && !argv.banranked && !argv.banunranked) {
```

Comment:
I think we only want `if (argv.ban)` here so we can have global bans as well as ranked or unranked specific bans?

Reply:
yes, this is in the next PR, refer to #129 for full commit list

Reply:
chewing the big one in small chunks :+1:

Reply:
Ok I've made this change on devel. I didn't add the text comment though so it will cause a minor conflict if you ever submit PR 2 again or something similar to it.